src/Controls/src/Core/Handlers/NavigationPage/NavigationPageHandler.Android.cs

```diff
@@ -148,12 +148,99 @@ void UpdatePadding()
 		{
 		}
 
-		void UpdateTitleColor()
+		void UpdateBarTextColor()
 		{
+			UpdateBarBackground();
 		}
-		void UpdateNavigationBarBackground()
+
+		void UpdateBarBackground()
```

Comment:
I'll move these to extensions in a later PR once I organize the navpage

Reply:
Can include a TODO comment here to avoid forget this?

Reply:
This whole file is a Todo. Once this all gets moved into interfaces and back to maui.core I'll fill this in more properly
https://github.com/dotnet/maui/issues/912